_test/bootstrap.sh

```diff
@@ -2,12 +2,6 @@
 # This installs a the prerequisites for a given GHC/Cabal config
 set -x
 set -e
-if [ ! -z "$GHCVER" ]; then
```

Comment:
no longer needed as now handled by `before_install`. Can make it a separate commit if desired, though it will be a bit tricky as there will be one commit that has `PATH` being set in `script/cabal`, then one that moves it up to `before_install`